provider/datagen/tests/data/json/units/info@1/und.json

```diff
@@ -0,0 +1,4150 @@
+    },
+    "ri-jp": {
+      "dimension": null,
+      "convert_info": 119
+    },
+    "rin": {
+      "dimension": null,
+      "convert_info": 120
+    },
+    "rod": {
+      "dimension": null,
+      "convert_info": 121
+    },
+    "sai": {
+      "dimension": null,
+      "convert_info": 122
+    },
+    "se-jp": {
+      "dimension": null,
+      "convert_info": 123
+    },
+    "second": {
+      "dimension": 41,
+      "convert_info": 124
+    },
+    "second-ampere": {
+      "dimension": 42,
+      "convert_info": null
+    },
+    "shaku": {
+      "dimension": null,
+      "convert_info": 125
+    },
+    "shaku-cloth": {
+      "dimension": null,
+      "convert_info": 126
+    },
+    "shaku-length": {
+      "dimension": null,
+      "convert_info": 127
+    },
+    "siemens": {
+      "dimension": null,
+      "convert_info": 128
+    },
+    "sievert": {
+      "dimension": null,
+      "convert_info": 129
+    },
+    "slug": {
+      "dimension": null,
+      "convert_info": 130
+    },
+    "solar-luminosity": {
+      "dimension": null,
+      "convert_info": 131
+    },
+    "solar-mass": {
+      "dimension": null,
+      "convert_info": 132
+    },
+    "solar-radius": {
+      "dimension": null,
+      "convert_info": 133
+    },
+    "square-meter": {
+      "dimension": 43,
+      "convert_info": null
+    },
+    "square-meter-per-square-second": {
+      "dimension": 44,
+      "convert_info": null
+    },
+    "square-revolution": {
+      "dimension": 45,
+      "convert_info": null
+    },
+    "steradian": {
+      "dimension": null,
+      "convert_info": 134
+    },
+    "stone": {
+      "dimension": null,
+      "convert_info": 135
+    },
+    "sun": {
+      "dimension": null,
+      "convert_info": 136
+    },
+    "tablespoon": {
+      "dimension": null,
+      "convert_info": 137
+    },
+    "teaspoon": {
+      "dimension": null,
+      "convert_info": 138
+    },
+    "tesla": {
+      "dimension": null,
+      "convert_info": 139
+    },
+    "therm-us": {
+      "dimension": null,
+      "convert_info": 140
+    },
+    "to-jp": {
+      "dimension": null,
+      "convert_info": 141
+    },
+    "ton": {
+      "dimension": null,
+      "convert_info": 142
+    },
+    "tonne": {
+      "dimension": null,
+      "convert_info": 143
+    },
+    "volt": {
+      "dimension": null,
+      "convert_info": 144
+    },
+    "watt": {
+      "dimension": null,
+      "convert_info": 145
+    },
+    "weber": {
+      "dimension": null,
+      "convert_info": 146
+    },
+    "week": {
+      "dimension": null,
+      "convert_info": 147
+    },
+    "week-person": {
+      "dimension": null,
+      "convert_info": 148
+    },
+    "yard": {
+      "dimension": null,
+      "convert_info": 149
+    },
+    "year": {
+      "dimension": 46,
+      "convert_info": 150
+    },
+    "year-person": {
+      "dimension": null,
+      "convert_info": 151
+    }
+  },
+  "unit_dimensions": [
```

Comment:
what are these sorted by? is it stable across CLDR releases?

Reply:
No it's not sorted and not stable, but the ordering is self-contained within this file. I think @younies already has an issue to make quantities an open enum.